docker/create_admin_user.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+set -x
```

Comment:
Is this left over from debugging, or does it do something special in conjunction with `-e`?

Reply:
it's a leftover, but on the other hand... it could be nice to set this flag in almost all of our shell scripts.  it helps with debugging and it's not like the output is particularly visible.  would you rather filter through garbage in the logs or bulid a new image every time you want to add this flag back in to diagnose a failure?  i could go either way i guess

Reply:
That could be helpful for when something goes wrong in production, but I feel like debugging type things should happen in dev.

I could go either way as well; so long as it's all-or-nothing. Maybe somebody else can weigh in on whether or not it should stay. @cherewaty @elrickvm @cowboyd

Reply:
>I feel like debugging type things should happen in dev
this makes sense, although i'm not sure we have a proper "dev" yet.  still, this isn't a dev environment.  let's go nothing on the all-or-nothing.